app/dash/Accounts/Add/AddHardware/index.js

```diff
@@ -43,9 +42,9 @@ class AddHardware extends React.Component {
             {tethered.length ? (
               tethered.map((signer, i) => {
                 return (
-                  <div className='addAccountItemOptionSetupFrame'>
+                  <div key={i + signer.id} className='addAccountItemOptionSetupFrame'>
```

Comment:
what's the linting rule that specifies this one?

Reply:
46:19  error  Missing "key" prop for element in iterator  `react/jsx-key`